src/scss/_brand.scss

```diff
@@ -0,0 +1,39 @@
+
+/// Output a color for a variant of a brand
+///
+/// @private
+
+// Lives inside brand file because of source order.
+@function _oLoadingGetColors($variant) {
+	@return oBrandGet($component: 'o-loading', $variables: 'default-color', $configure-for: $variant);
```

Comment:
I don't _think_ this is needed, can the `oBrandGet` call be in `src/scss/_variables.scss`?